Sources/Control/Volume.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Volume.swift
+//  ControlKit
+//
+
+import MediaPlayer
+
+public extension Control {
+    
+    /// Control the system volume using the underlying `MPVolumeView`.
+    @MainActor
+    enum Volume {
+        
+        /// Subscribe to `volume` via ``VolumeController.volume`` `@Published` property.
+        public static var volume: Float {
+            get { MPVolumeView.volume }
+            set { MPVolumeView.volume = newValue }
+        }
+        
+        /// Subscribe to `isMuted` via ``VolumeController.isMuted`` `@Published` property.
+        static var isMuted = false
+        
+        private static var mutedValue: Float = 0
+        
+        public static func increaseVolume(_ amount: Float = 0.1) {
+            volume += amount
+        }
+        
+        public static func decreaseVolume(_ amount: Float = 0.1) {
+            volume -= amount
```

Comment:
Please correct me if I'm wrong 🐼, seems like if `isMuted` set to `true`, any changes to `Volume.volume` would unmute source, but `isMuted` becomes out of sync 👀

Reply:
# I totally missed this 🫨

Reply:
Let me know what you think of the fixes in [this fixup commit](https://github.com/superturboryan/ControlKit/pull/1/commits/894d15bb7699b2b27effb6c1aab665292f054221) @erikbasargin 🙋🏻‍♂️